administrator-manual/en/web_server.rst

```diff
@@ -0,0 +1,262 @@
+.. _web-server-section:
+
+.. index:: virtual hosts
+   single: HTTP
+   single: HTTPS
+
+==========
+Web server
+==========
+
+.. note::
+
+    This chapter describes the features of the *Web server* application,
+    available in the new Server Manager.
+
+After |product| has been installed the :guilabel:`Web server` application is
+already available. It configures and starts the Apache HTTP web server.
+
+The :guilabel:`Web server` application provides the following features:
+
+- Apache HTTP web server
+- Integration with system certificates for HTTPS
+- Hosting of multiple web sites 
+- HTTP :index:`reverse proxy`
+- :index:`PHP` scripting language to run web applications
+- :index:`FTP server`
+
+The following sections describe the pages of the :guilabel:`Web server`
+application. Some of them require additional software components that are
+automatically downloaded, installed and configured when they are required for
+the first time.
+
+
+Web server dashboard
+====================
+
+The :guilabel:`Web server dashboard` page shows the current web server status
+and statistics. It also lists the additional components installed on the system.
+
+The default Apache configuration serves the contents of :file:`/var/www/html`
+and is capable of executing PHP scripts by running them on the
+:guilabel:`Default web stack`.
+
+.. _webserver-settings-section:
+
+Settings
+========
+
+The :guilabel:`Settings` page allows to change the PHP configuration parameters
+for resources allocation (e.g. script maximum memory and execution time).
+
+Changes affect the global PHP configuration: as such they are valid for both web
+applications and command line scripts, unless they are overridden by some means.
+
+PHP settings can also be adjusted for a specific web site from the
+:guilabel:`Virtual hosts` page (see also :ref:`webserver-phpversions-section`),
+or overridden with a custom configuration file.
```

Comment:
I'd remove this sentence or add an example for it (or a pointer to another section)